packages/main/src/backend/import/downloadChromium.ts

```diff
@@ -34,21 +37,85 @@ export default async function downloadChromium(installPath: string, onProgress?:
 
   logger.log(`Browser: ${Browser.CHROMIUM}, Platform: ${platform}, Tag: stable, BuildId: ${buildId}`);
 
-  downloadProm = install({
+  // Create proxy agent if system proxy is configured
+  const proxyAgent = createProxyAgent();
+
+  // Store original global agents
+  const originalHttpAgent = http.globalAgent;
+  const originalHttpsAgent = https.globalAgent;
+
+  // Set proxy agent globally if available
+  if (proxyAgent) {
+    http.globalAgent = proxyAgent;
+    https.globalAgent = proxyAgent;
+    logger.log('Using proxy agent for Chromium download');
+  }
```

Comment:
Since there is no usage of the proxy (it configured globally), you should create a file for the proxy configurations.
That file will expose 3 functions:
1. `initProxyIfNeeded` will first run an internal function `if (!isProxyNeedConfiguration()) { return; }` and then will run all the configurations here including the `http.globalAgent` assignment.
1. `tearDownProxy` to restore the original configurations on `finally` block.
1. `getConfigurations` or something like this, because on `bankScraper` file you repeated the same code of reading env vars.